### PR TITLE
Create environment for testing: DDFLSBP-84 (systemfeedback)

### PR DESCRIPTION
**Pull request by kasperbirch1**
#### Link to issue
https://reload.atlassian.net/browse/DDFLSBP-84
https://github.com/danskernesdigitalebibliotek/dpl-react/pull/606

#### Description

This PR introduces necessary translations corresponding to the changes in https://github.com/danskernesdigitalebibliotek/dpl-react/pull/606 and also updates the dpl-react package.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -122,15 +122,33 @@
...
+            ],
+            "drush/Commands/contrib/{$name}": [
+                "type:drupal-drush"
+            ],
+            "web/modules/custom/{$name}": [
+                "type:drupal-custom-module"
+            ],
+            "web/themes/custom/{$name}": [
+                "type:drupal-custom-theme"
+            ]
```
Comment (spaceo) [inline composer.json:151]: No need to reformat these lines I think, right?

**Review by spaceo — CHANGES_REQUESTED** [review]
Looks fine. One comment...

File: composer.json (hunk 1 above)
Comment (kasperbirch1) [reply composer.json:151]: No it was just done automatically so I thought it would happen to the next one too?

Comment (kasperbirch1) [reply composer.json]: I have changed it back now

**Review by spaceo — APPROVED** [review]
LGTM 👍

**Pull request merged**